src/lib.rs

```diff
@@ -20,9 +20,12 @@
 //! and its derivatives.
 //!
 //! To minimize dependencies, some functions are feature-gated. To generate
-//! random keys or to re-randomize a context object, compile with the "rand"
-//! feature. To de/serialize objects with serde, compile with "serde".
-//! **Important**: `serde` encoding is **not** the same as consensus encoding!
+//! random keys or to re-randomize a context object, compile with the `rand`
+//! feature. Unless you _really_ do not want the `rand` dependency consider
+//! using the `rand` feature, creating contexts without randomizing can leave
+//! you open to side channel attacks. To de/serialize objects with serde,
```

Comment:
"Can leave you open to sidechannel attacks" is definitely too strong. I would invert this whole paragraph to say something like "If you are willing to use a `rand` dependency, we have enabled an additional defense-in-depth sidechannel protection for our context objects, which re-blinds certain operations on secret key data".

Reply:
Use your suggested text, almost verbatim. The relevant section now reads
```
//! To minimize dependencies, some functions are feature-gated. To generate
//! random keys or to re-randomize a context object, compile with the `rand`
//! feature. If you are willing to use the `rand` dependency, we have enabled an
//! additional defense-in-depth sidechannel protection for our context objects,
//! which re-blinds certain operations on secret key data. To de/serialize
//! objects with serde, compile with "serde". **Important**: `serde` encoding is
//! **not** the same as consensus encoding!
```